src/core/scr_globals.hpp

```diff
@@ -8,7 +8,7 @@ namespace big::scr_globals
 	static inline const script_global gsbd_fm(1845281);
 	static inline const script_global gsbd_kicking(1877042);
 	static inline const script_global gsbd_fm_events(1916617);
-	static inline const script_global gsbd_block_c(2657971);
+	static inline const script_global gsbd_block_c(2652592);
```

Comment:
there is one more global duplicate named `globalplayer_bd` which should also be changed. Infact i dont see where we are using this global `gsbd_block_c`

Reply:
LGTM